pkg/ui/workspaces/db-console/src/views/reports/containers/network/index.tsx

```diff
@@ -251,94 +277,96 @@ export class Network extends React.Component<NetworkProps, INetworkState> {
     }
 
     // legend is just a quick table showing the standard deviation values.
-    return [
-      <Sort
-        onChangeCollapse={this.onChangeCollapse}
-        collapsed={collapsed}
-        sort={sortParams}
-        filter={filter}
-        onChangeFilter={this.onChangeFilter}
-        deselectFilterByKey={this.deselectFilterByKey}
-      />,
-      <div className="section">
-        <Legend
-          stddevMinus2={stddevMinus2}
-          stddevMinus1={stddevMinus1}
-          mean={mean}
-          stddevPlus1={stddevPlus1}
-          stddevPlus2={stddevPlus2}
-          noConnections={noConnections}
+    return (
+      <Fragment>
+        <Sort
+          onChangeCollapse={this.onChangeCollapse}
+          collapsed={collapsed}
+          sort={sortParams}
+          filter={filter}
+          onChangeFilter={this.onChangeFilter}
+          deselectFilterByKey={this.deselectFilterByKey}
         />
-        {latencyTable}
-      </div>,
-    ];
+        <div className="section">
+          <Legend
+            stddevMinus2={stddevMinus2}
+            stddevMinus1={stddevMinus1}
+            mean={mean}
+            stddevPlus1={stddevPlus1}
+            stddevPlus2={stddevPlus2}
+          />
+          {latencyTable}
+        </div>
+      </Fragment>
+    );
   }
 
   getSortParams = (data: Identity[]) => {
```

Comment:
This function is quite lengthy with a lot of nested array processing, can you leave a comment summarizing what it's doing?